ffs/scheduler/types.go

```diff
@@ -1,59 +0,0 @@
-package scheduler
-
-import (
-	"context"
-	"errors"
-
-	"github.com/ipfs/go-cid"
-	"github.com/textileio/powergate/ffs"
-)
-
-var (
-	// ErrNotFound is returned when an item isn't found on a Store.
-	ErrNotFound = errors.New("item not found")
-)
-
```

Comment:
We don't expect more than one impl of this stores.
At most, we can give the stores different implementations of `datastore.Datastore` (if want to save in other databases, etc), that's no problem.